node/service/src/chain_spec.rs

```diff
@@ -273,7 +286,7 @@ pub fn cere_dev_development_config() -> Result<CereDevChainSpec, String> {
 		None,
 		Some(DEFAULT_PROTOCOL_ID),
 		None,
-		None,
+		Some(cere_dev_native_chain_spec_properties()),
```

Comment:
Problem is, this requires a reset of DevNet since the properties are initialized in the the genesis block.

Reply:
It doesn't. This code refers only to `dev` chain spec, the one that is used when running `./target/cere --dev`